src/app/event/new-scheduled-event/new-scheduled-event.component.ts

```diff
@@ -586,7 +590,7 @@ export class NewScheduledEventComponent implements OnInit {
       .pipe(
         concatMap((e: Environment[]) => {
           this.environments = e;
-          return from(e);
+          return e
```

Comment:
Is this intentional? The removal of `from(e)` 
IIRC that returns now the Environment instead of an Observable<Environment>

Reply:
This did not break the functionality. Thus i figured this is unnecessary